assets/css/climatehub.css

```diff
@@ -7,3 +7,21 @@
 #someId {
   width: 100%;
 }
+
+.home-Page-Banner-Bottom-Left{
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.home-Page-Banner{
+  height: 60vh;
+}
+
+.home-page-banner-width-title{
+  width: 46%;

```

Comment:
If it doesn't need to be 46% exactly, you could add the class `w-50` to the h1 element and it will set the width to be 50%. Same thing for line 26.